draft-dahlberg-ll-quantum-03.xml

```diff
@@ -254,8 +254,11 @@ is to manage re-tries in producing entanglement at the the physical layer.
                A binary value encoding the integer 'n' represents the fidelity 'n' divided by (2^16-1).
              </t>
              <t>
-               Max Time (16 bits): The maximum time in milliseconds that the higher layer is willing to wait for the request to be fulfilled.
-               Represented as a binary16 float as specified in <xref target="binary16"/>.
+               Time unit (tu) (2 bits): The time units used for specifying Max Time, where (00, 01, 10) each indicate (micro-seconds, milliseconds, seconds) respectively and 11 is unused.
+             </t>
+             <t>
+               Max Time (16 bits): The maximum time in the time units specified above that the higher layer is willing to wait for the request to be fulfilled.
```

Comment:
Max Time and Time unit together make up 16 bits, so Max Time here needs to be 14 bits and the header diagram also needs to be updated to make sure Max Time is split into Max Time and tu

Reply:
Done